src/bun.js/bindings/JSValue.zig

```diff
@@ -491,6 +491,18 @@ pub const JSValue = enum(i64) {
         return result;
     }
 
+    // ECMA-262 20.1.2.5
```

Comment:
https://tc39.es/ecma262/#sec-number.issafeinteger is a much more stable reference if this is the algorithm you were referencing

Reply:
> 20.1.2.5 Object.entries ( O )

Reply:
will update 👍